include/geom/cell_hex.h

```diff
@@ -180,6 +180,9 @@ inline
 Hex::Hex(const unsigned int nn, Elem* p, Node** nodelinkdata) :
   Cell(nn, Hex::n_sides(), p, _elemlinks_data, nodelinkdata)
 {
+  // Make sure the interior parent isn't undefined
+  if (LIBMESH_DIM > 3)
```

Comment:
Never, but I figure there's no reason to preclude it.  Twenty years from now when we're solving on tesseracts in space-time meshes you'll thank me.